plaidml/BUILD

```diff
@@ -36,6 +36,7 @@ plaidml_cc_library(
         "//pmlc/target/demo",
         "//pmlc/target/intel_gen",
         "//pmlc/target/x86",
+        "//pmlc/rt/vulkan",
```

Comment:
This shouldn't be added here, it should be part of the intel_gen target.

Reply:
WAI -- experimental code